docs/en/ingest-management/fleet/fleet-settings.asciidoc

```diff
@@ -20,8 +20,13 @@ Configure {fleet} settings to apply global settings to all {agent}s enrolled in
 *{fleet-server} hosts*
 
 | The URLs your {agent}s will use to connect to a {fleet-server}. This setting
-is required. On self-managed clusters, you must specify one or more URLs. On
-{ecloud}, this field is populated automatically.
+is required. On self-managed clusters, you must specify one or more URLs.
+
+On {ecloud}, this field is populated automatically. If you are using
+Azure Private Link, GCP Private Service Connect, or AWS PrivateLink
+and enrolling the Elastic Agent with a private link URL, you need to
```

Comment:
```suggestion
and enrolling the {agent} with a private link URL,
```

Reply:
"you need to" seems like extra words here, but if you feel it's really necessary for emphasis, you can ignore my suggestion.